pkg/vulnsrc/osv/osv.go

```diff
@@ -212,12 +218,16 @@ func (o OSV) parseAffected(entry Entry, vulnIDs, aliases, references []string) (
 
 	uniqAdvisories := map[string]Advisory{}
 	for _, affected := range entry.Affected {
-		ecosystem := convertEcosystem(affected.Package.Ecosystem)
-		if ecosystem == vulnerability.Unknown {
+		bkt, err := o.convertEcosystem(affected.Package.Ecosystem)
+		if err != nil {
+			// Skip unsupported ecosystems
+			continue
+		}
+		if lo.IsNil(bkt) { // `bkt == nil` does not work here because of the interface type
```

Comment:
nit:
```suggestion
			if err != nil || lo.IsNil(bkt) { // `bkt == nil` does not work here because of the interface type
			// Skip unsupported ecosystems
			continue
		}
```

Reply:
Thanks. I forgot to remove it.
https://github.com/aquasecurity/trivy-db/pull/572/commits/953d63d577db8fd096242647e55e8fe5b3e87ab2